doc/src/conf.py

```diff
@@ -363,26 +363,20 @@
 
 # Additional stuff for the LaTeX preamble.
 # Tweaked to work with XeTeX.
+latex_engine = 'xelatex'
 latex_elements = {
-    'babel':     '',
-    'fontenc': r'''
-% Define version of \LaTeX that is usable in math mode
-\let\OldLaTeX\LaTeX
-\renewcommand{\LaTeX}{\text{\OldLaTeX}}
-
+    'babel': r'\usepackage[english]{babel}',
+    'fontpkg': r'''
 \usepackage{bm}
-\usepackage{amssymb}
```

Comment:
This `\usepackage{bm}` seems to be there to allow `\bm` mark-up to obtain bold math.  Which is good, but I don't know if it is documented in Contributor guide and searching for it in the produced LaTeX file from `make latex`, I do not find a single example so I guess there are none in soures (it is easier to seach in monolithic LaTeX than across multiple sources via `grep -r`). Ping @asmeurer

Reply:
From the git blame, this comes from https://github.com/sympy/sympy/pull/1493

Reply:
I traced the `\usepackage{bm}` seemingly first appearance in 
bed2d9332af410c67.  It was used before in b43b9dc256213 but for `pngmath_latex_preamble` which is not quite same thing (and the line `pngmath_divpng_args = ['-gamma 1.5','-D 110']` had a typo probably it should have been `pngmath_dvipng_args` as the binary is `dvipng` not `divpng` but it is a bit late in 2024 to review a 2009 commit...).  Anyhow in both cases the only reason for this is to allow `\bm` mark-up in math.  But from examining the (huge) LaTeX file, I see that there is currently not a single use of `\bm` in Sympy documentation.

Build succeeds after removal of `\usepackage{bm}`. I have no strong opinion on whether to keep it or not.  Many people use `\mathbf` in math with the hope it does what actually `\bm` is able of doing, not `\mathbf`.  As long as documentation does not mention `\bm` as recommended to contributors, there is no strong reason to keep it, but it is not a matter of primary importance.

Reply:
A commit has since removed this `\usepackage{bm}`.